h2/src/main/org/h2/engine/Database.java

```diff
@@ -1123,6 +1123,11 @@ public void removeMeta(Session session, int id) {
         }
     }
 
+    /**
+     * Release IDs.
```

Comment:
They are not identifiers of releases, name of parameter and its comment are correct, this comment is not.

Reply:
OK. I don't catch the code logic idea.
Drop comment.